### PR TITLE
Initial pass at watchman builds.

### PR DESCRIPTION
**Pull request by kwlzn**

### CODE REVIEW

File: build-watchman.sh [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"`
+DIRPATH=$(pwd -P)
+WATCHMAN_DEST_DIR="${DIRPATH}/build-support/bin/watchman/${PLATFORM}/${ARCH}/${WATCHMAN_VERSION}"
+BUILD_DIR="watchman_build.${DATE}"
+mkdir -p $BUILD_DIR
+
+PCRE_DIR="pcre-${PCRE_VERSION}"
+PCRE_TARBALL="${PCRE_DIR}.tar.gz"
+PCRE_URL="http://downloads.sourceforge.net/project/pcre/pcre/${PCRE_VERSION}/${PCRE_TARBALL}"
```
Comment (stuhood) [inline build-watchman.sh]: Mind including a shasum check for this one? sourceforge ain't what it used to be.

**Comment by stuhood** [comment]
I think you'll want to populate at least some of the OSX versions with symlinks to this binary... I smoketested that it works on 10.9.x at least.

**Comment by stuhood** [comment]
#shipit

**Comment by kwlzn** [comment]
added a sha1 checksum and some symlinks.

**Pull request merged**